apps/site/src/app/admin/layout/AdminSidebar.tsx

```diff
@@ -33,10 +33,37 @@ function AdminSidebar() {
 		});
 	}
 
+	// TODO: change access to role "Volunteer Reviewer"
+	if (isApplicationManager(roles)) {
+		navigationItems.splice(2, 0, {
+			type: "link",
+			text: "Volunteer Applications",
+			href: "/admin/applicants/volunteers",
+		});
+	}
+
+	// TODO: change access to role "Mentor Reviewer"
+	if (isApplicationManager(roles)) {
+		navigationItems.splice(2, 0, {
+			type: "link",
+			text: "Mentor Applications",
+			href: "/admin/applicants/mentors",
+		});
+	}
+
+	// TODO: change access to role "Hacker Reviewer"
+	if (isApplicationManager(roles)) {
+		navigationItems.splice(2, 0, {
+			type: "link",
+			text: "Hacker Applications",
+			href: "/admin/applicants/hackers",
+		});
+	}
```

Comment:
Suggestion: not sure how tricky this will be with the more specific authorization, but given the path structure we previously decided, I think it would make more sense to nest these within Applicants, see example for [expandable link groups in Side navigation](https://cloudscape.design/components/side-navigation/?tabId=playground&example=with-expandable-link-groups).